.github/workflows/swift-toolchain.yml

```
@@ -2306,6 +2306,16 @@ jobs:
       - run: swift test -Xswiftc -DENABLE_TESTING
         working-directory: ${{ github.workspace }}/SourceCache/swift-win32
 
+      - name: Test that sourcekit-lsp initializes
```

I would just do `SourceKit Validation`

```suggestion
      - name: SourceKit-LSP Validation
```